exporter/collector/README.md

```diff
@@ -192,6 +200,123 @@ following proxy environment variables:
 If set at Collector start time then exporters, regardless of protocol,
 will or will not proxy traffic as defined by these environment variables.
 
+### Logging Exporter
+
+The logging exporter processes OpenTelemetry log entries and exports them to GCP Cloud Logging. Logs can be collected using one 
+of the opentelemetry-collector-contrib log receivers, such as the [filelogreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+Log entries must contain any Cloud Logging-specific fields as a matching OpenTelemetry attribute (as shown in examples from the 
+[logs data model](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#google-cloud-logging)). 
+These attributes can be parsed using the various [log collection operators](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/operators/README.md#what-operators-are-available) available upstream.
+
+For example, the following config parses the [HTTPRequest field](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#HttpRequest) from Apache log entries saved in `/var/log/apache.log`. 
+It also parses out the `timestamp` and inserts a non-default `log_name` attribute and GCP [MonitoredResource](https://cloud.google.com/logging/docs/reference/v2/rest/v2/MonitoredResource) attribute.
+
+```yaml
+receivers:
+  filelog:
+    include: [ /var/log/apache.log ]
+    start_at: beginning
+    operators:
+      - id: http_request_parser
+        type: regex_parser
+        regex: '(?m)^(?P<remoteIp>[^ ]*) (?P<host>[^ ]*) (?P<user>[^ ]*) \[(?P<time>[^\]]*)\] "(?P<requestMethod>\S+)(?: +(?P<requestUrl>[^\"]*?)(?: +(?P<protocol>\S+))?)?" (?P<status>[^ ]*) (?P<responseSize>[^ ]*)(?: "(?P<referer>[^\"]*)" "(?P<userAgent>[^\"]*)")?$'
+        parse_to: attributes["gcp.http_request"]
+        timestamp:
+          parse_from: attributes["gcp.http_request"].time
+          layout_type: strptime
+          layout: '%d/%b/%Y:%H:%M:%S %z'
+    converter:
+      max_flush_count: 100
+      flush_interval: 100ms
+
+exporters:
+  googlecloud:
+    project: my-gcp-project
+    retry_on_failure:
+      enabled: true
+    log:
+      default_log_name: apache-access-log
+
+processors:
+  memory_limiter:
+      check_interval: 1s
+      limit_mib: 4000
+      spike_limit_mib: 800
+  batch:
+    send_batch_max_size: 200
+    send_batch_size: 200
+  resource:
+    attributes:
+      - key: cloud.platform
+        action: insert
+        value: gcp_compute_engine
```

Comment:
```suggestion
  resourcedetection:
    detectors: [gce, gke]
    timeout: 10s
```
Can we suggest using the resource detectors instead of hard-coding?  Those should set cloud.platform correctly.

Reply:
Yup that sounds good